hash/mac_flop.xml

```diff
@@ -29,6 +29,40 @@
 			</dataarea>
 		</part>
 	</software>
+	
+	<!-- S/N: Z93350-001A -->
+	<software name="clarisw">
+		<description>ClarisWorks (Swe)</description>
```

Comment:
This is another case where having the target market in the set name is probably more useful than not - ClarisWorks was released in a large number of variants, including specific UK and Australian English versions.